TwitchPlugin/TwitchPlugin/Properties/launchSettings.json

```diff
@@ -0,0 +1,12 @@
+{
+  "profiles": {
+    "TwitchPluginWin": {
+      "commandName": "Executable",
+      "executablePath": "C:\\Program Files\\Logi\\LogiPluginService\\LogiPluginService.exe"
+    },
+    "TwitchPluginMac": {
+      "commandName": "Executable",
+      "executablePath": "/Applications/Loupedeck.app/Contents/MacOS/Loupedeck"

```

Comment:
The path has changed. The app is now in `/Applications/Utilities/LogiPluginService.app`.